src/components/search/FilterMenu.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  Box,
+  Slider,
+  SliderThumb,
+  Stack,
+  styled,
+  TextField,
+  Typography,
+} from '@mui/material';
+import { ControllerRenderProps } from 'react-hook-form';
+
+import { SubTitle } from '@/components/common/Title';
+import { FilterFormType } from '@/types/search';
+import { getTextBySubTitle } from '@/utils/helper';
+
+interface FilterMenuProps {
+  subTitle: '여행 기간' | '여행 경비';
+  // 해당부분을 ControllerRenderProps<FilterFormType, 'maxDays' | 'maxCost'>; 로 사용하려 했는데 타입에러 발생
+  minState:
+    | ControllerRenderProps<FilterFormType, 'minDays'>
+    | ControllerRenderProps<FilterFormType, 'minCost'>;
+  maxState:
+    | ControllerRenderProps<FilterFormType, 'maxDays'>
+    | ControllerRenderProps<FilterFormType, 'maxCost'>;
+  minValue: number;
+  maxValue: number;
+  steps: number;
+}
+
+const FilterMenu = ({
+  subTitle,
+  minState,
+  maxState,
+  minValue,
+  maxValue,
+  steps,
+}: FilterMenuProps) => {
+  const { minPlaceholder, maxPlaceholder, helperText } = getTextBySubTitle(
+    subTitle,
+    minState.value ? minState.value : '',
+    maxState.value ? maxState.value : '',
```

Comment:
proposal;

아래처럼 더 간결하게 작성할 수 있을 것 같습니다.
```suggestion
    minState.value || '',
    maxState.value || '',
```